src/systemctl/systemctl.c

```diff
@@ -4907,10 +4907,7 @@ static int print_property(const char *name, sd_bus_message *m, const char *conte
                                 return bus_log_parse_error(r);
 
                         while ((r = sd_bus_message_read(m, "(ss)", &type, &path)) > 0)
-                                if (arg_value)
-                                        puts(path);
-                                else
-                                        printf("Listen%s=%s\n", type, path);
+                                print_prop(name, "%s (%s)", path, type);
```

Comment:
Hmm, I don't think this is an improvement... This output is much harder to use in scripts, grep, etc. Why not go the other way, and fix the query code so that `systemctl show -p ListenStream` does the expected thing?

Reply:
Hmm... `systemctl show -p` does not take unit settings, but takes dbus properties. E.g., it supports `*USec` instead of `*Sec`:
```
$ systemctl show -p RestartUSec systemd-journal.service
RestartUSec=100ms
$ systemctl show -p RestartSec systemd-journal.service
```
So, I think making `-p` support to map from a unit settings to the relevant dbus property may be a good idea, but it is not a scope of this PR.

Reply:
Makes sense.